about.html

```diff
@@ -0,0 +1,135 @@
+          <li><a href="#"><i class='bx bxl-twitter bx-sm'></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <nav class="d-flex">
+        <a class="burger" href="#">
+          <i class='bx bx-menu bx-md'></i>
+        </a>
+        <a href="index.html">
+          <h1 class="frank-logo" >Musica</h1>
+        </a>
+        <ul class="desktop-links">
+          <li class="primary-color"><a href="about.html">About</a></li>
+          <li ><a class="secondary-color" href="#">Program</a></li>
+          <li ><a class="secondary-color" href="#">Join</a></li>
+          <li><a class="secondary-color" href="#">Sponsor</a></li>
+          <li><a class="secondary-color" href="#">News</a></li>
+          <li class="primary-color spec"><a href="#">Campaign</a></li>
+        </ul>
+      </nav>
+      <div class="menu-container">
+        <div class="close-btn">
+          <i class="bx bx-x bx-lg closeIcon"></i>
+        </div>
+        <ul>
+          <li><a class="side-link" href="index.html">Home</a></li>
+          <li><a class="side-link" href="about.html">About</a></li>
+          <li><a class="side-link" href="#aboutme">Program</a></li>
+          <li><a class="side-link" href="#contact">Join</a></li>
+          <li><a class="side-link" href="#contact">Sponsor</a></li>
+          <li><a class="side-link" href="#contact">Campaign</a></li>
+        </ul>
+      </div>
+      <div class="text">
+        <p class="greeting-text primary-color">"Hi Music Enthusiats"</p>
+        <h2 class="primary-color"><span class="primary-color">Franky</span><span class="primary-color"> Musical</span></h2>
+        <h3 class="primary-color">Gobal Concert <small>2022</small></h3>
+      </div>
+      <div class="secondary-color">
+        <p class="hero-text sub-text">
+          The Global Concert brings together the community of experts, musician, and academics who comprise the Creative
+          Commons affiliate network in a different country every two.
+          This year, we hope to expand our invitation list including organizations and individuals.
+        </p>
+        <p class="hero-text contact-text">
+          Please contact us via Email for any further questions about Global Concert 2022
+        </p>
+        <p class="contact-email">
+          frankyglobalc@globalconcert.com
+        </p>
+      </div>
+    </header>
+  </div>
+  <main>
+    <section class="about">
+      <h4 class="program-text secondary-color">Franky Global Musical Concert 2022 logo</h4>
+      <hr class="primary-color line">
+      <p class="hero-text contact-text">
+        This Logo was designed by Graphixray_Designz back at the February 2016 summit. It was accepted after winning
+        an award of best logo design. Graphixray_Designz was given a noble prize.
+      </p>
+      <div class="concert-logo">
```

Comment:
- [ ] The section is under, the div please re-adjust the CSS.

![section](https://user-images.githubusercontent.com/61562626/142412899-909cc018-e43b-45f1-aa65-4868b0f9f663.jpeg)

Reply:
- [ ] Addressed this but it in the footer you should add a logo
check the original
![footer](https://user-images.githubusercontent.com/44745855/142426851-96034d08-1602-4be3-beeb-708d30b7e5a3.png)